imgaug/augmenters/geometric.py

```diff
@@ -2069,7 +2111,7 @@ class PerspectiveTransform(meta.Augmenter):
 
     """
 
-    def __init__(self, scale=0, keep_size=True, name=None, deterministic=False, random_state=None):
+    def __init__(self, scale=0, keep_size=True, name=None, cval=0, mode='constant', deterministic=False, random_state=None):
```

Comment:
Can you move `cval` and `mode` before `keep_size`, so that it is `self, scale=0, cval=0, mode="constant", keep_size=True, ...`? That would be more in line with the order of arguments that other augmenters use. (Breaks a bit more old code though, but seems acceptable here.)